sdk/src/pubkey.rs

```diff
@@ -13,3 +13,21 @@ pub fn log_pubkey(pubkey: &Pubkey) {
     #[cfg(not(target_os = "solana"))]
     core::hint::black_box(pubkey);
 }
+
+pub fn find_program_address(seeds: &[&[u8]], program_id: &Pubkey) -> (Pubkey, u8) {
+    let mut bytes = [0; 32];
+    let mut bump_seed = u8::MAX;
+    let result = unsafe {
+        crate::syscalls::sol_try_find_program_address(
+            seeds as *const _ as *const u8,
+            seeds.len() as u64,
+            program_id as *const _ as *const u8,
+            &mut bytes as *mut _ as *mut u8,
+            &mut bump_seed as *mut _,
+        )
+    };
+    match result {
+        crate::entrypoint::SUCCESS => (bytes, bump_seed),
+        _ => panic!("PDA cannot be found"),
```

Comment:
Should we use the same message from `solana_program`, e.g., `"Unable to find a viable program address bump seed"`?

Reply:
I think we should keep error logs consistent